### PR TITLE
Made "Enter" key advance workflow in valid market.

### PR DESCRIPTION
**Pull request by michelleinez**
Github issue #1336

[Clubhouse Story](https://app.clubhouse.io/augur/story/5878)

## Submitter checklist
- [ ] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [ ] Screenshot (If applicable)

## Reviewer Checklist
- [x] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by bthaile** [comment]
Resolved conflicts on branch.

**Comment by bthaile** [comment]
I'm not a big fan of using bug fixes from fork of seadragon, it's out of my normal review process.

help me out and teach me how to pull and test your code without having to clone your repo.

File: src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx [hunk 1]
```diff
@@ -403,6 +412,7 @@ export default class CreateMarketOutcome extends Component {
             onChange={(e) => {
               updateNewMarket({ detailsText: e.target.value })
             }}
+            onKeyPress={e => keyPressed(e)}
           />
```
Comment (bthaile) [inline src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx]: Let's not capture keyPressed for additional details. The use will probably want to format the text which will include carriage returns. 

They'll have to tab out of the textarea in order to hit <enter> and advance.

**Comment by tinybike** [comment]
Hm, pressing enter doesn't seem to work everywhere on the date selection page (it gets swallowed by the expiration date and expiration time dropdowns).

**Pull request merged**